Use custom banners instead of deprecated admonitions (#1159)
iotaledger/iota-wiki: banners/iota-streams-deprecated.mdx

> This is documentation for Streams, which is no longer actively maintained.

Reviewer: ```suggestion
The for Streams is no longer actively maintained.
```

Author: Something is missing here 😆 Do you mean
```suggestion
The documentation for Streams is no longer actively maintained.
```
?
Does that explain well enough that the library will not be maintained? Or only the docs? 😅

Reviewer: It's pretty ambiguous in both cases. How about:

```suggestion
Streams and this documentation are no longer actively maintained.
```
